src/Microsoft.AspNet.Authentication.OAuthBearer/OAuthBearerAuthenticationOptions.cs

```diff
@@ -140,15 +140,10 @@ public TokenValidationParameters TokenValidationParameters
             {
                 return _tokenValidationParameters;
             }
-
+            [param: NotNull]
```

Comment:
FYI, this property will be simplified in another PR: https://github.com/aspnet/Security/pull/276/files#diff-377cf3927416f05a94fa6f3a47471549R114